### PR TITLE
fix(r-date-input): fix the input event prop for date only type

### PR DESCRIPTION
**Pull request by dariashka**
### What was a problem?

For `type` of Date Input it should `@input` with the date only, without any time and timezone markers.

### How this PR fixes the problem?

Now it looks like `@input('2022-01-01')`

### CODE REVIEW

File: packages/recomponents/src/components/r-date-input/r-calendar-manager.vue [hunk 1]
```diff
@@ -256,6 +257,7 @@
                     return;
                 }
                 this.$emit('input', date);
+                this.$refs.datePicker.hidePopover();
```
Comment (p3yman) [inline packages/recomponents/src/components/r-date-input/r-calendar-manager.vue:260]: Isn't it better to have an option for closing on select with the default value being true?

File: packages/recomponents/src/components/r-date-input/r-date-input.vue [hunk 2]
```diff
@@ -193,8 +193,10 @@
                         start: moment(date.start).utc().tz(this.timezone, true),
                         end: moment(date.end).utc().tz(this.timezone, true),
                     };
+                } else if (this.type === DateInputType.date) {
```
Comment (cesarlevel) [inline packages/recomponents/src/components/r-date-input/r-date-input.vue:196]: Will this change bring issues to Recomm? that is probably expecting the UTC time? I wonder if would it be possible to have this `if` to include something like `this.type === DateInputType.date && !this.timezone)` the only issue with this is that if we don't include the timezone the component returns `undefined`.

Comment (dariashka) [reply packages/recomponents/src/components/r-date-input/r-date-input.vue:196]: I'm preparing the PR for it. We don't use type `date` too much.

Comment (dariashka) [reply packages/recomponents/src/components/r-date-input/r-date-input.vue:204]: IMO the `date` type is not supposed to return the date with time value. It should return absolute only date value.

**Review by cesarlevel — APPROVED** [review]

**Pull request merged**